components/private/ThumbTrack/src/MDCThumbTrack.m

```diff
@@ -104,10 +104,24 @@ - (void)drawRect:(CGRect)rect {
   if (_numDiscreteDots >= 2) {
     CGContextRef contextRef = UIGraphicsGetCurrentContext();
 
-    CGRect circleRect = CGRectMake(0, 0, self.bounds.size.height, self.bounds.size.height);
+    // The dot should be:
+    // +  (1, 1) at trackHeight == 1
+    // +  (2, 2) at trackHeight == 2
+    // +  (2, 2) at trackHeight == 4
+    // +  (2.5, 2.5) at trackHeight == 5
+    // +  (3, 3) at trackHeight == 6
+    // +  ... and so on.
```

Comment:
These examples are very useful, but adding a more general explanation of what is happening here would be also helpful so the reader doesn't have to derive the logic from the examples (this needs to be wordsmithed a bit, but something like "for trackHeight < 4, the tick marks are dots with a 2pt diameter. For track heights > 4, the tick marks become rounded rects with height of trackHeight/2"). Also, the size values for trackHeight > 4 are not correct – at trackHeight 6, the dot is not (3,3), but (2,3).

Reply:
Done, please take a look at the latest version.